### PR TITLE
PSTN fix ups

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

- Updates the video gallery to not use the dominant speaker logic for the `calling...` state participants
- updates sample app to parse out special characters from phone number you are calling.
# Why

Helps with performance when multiple people are joining a call we aren't using the existing participant speaking logic to try and sort the joining participants

# How Tested

Ran locally in samples

### CODE REVIEW

File: samples/Calling/src/app/views/HomeScreen.tsx [hunk 1]
```diff
@@ -190,7 +190,8 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                        * We need to pass in the formatting for the phone number string in the onChange handler
                        * to make sure the phone number is in E.164 format.
                        */
-                      const phoneNumber = '+' + newValue?.replace('[^0-9]', '');
+                      const phoneNumber =
```
Comment (anjulgarg) [inline samples/Calling/src/app/views/HomeScreen.tsx:193]: How about
`newValue?.replace(/\D/g, '')`

File: samples/Calling/src/app/views/HomeScreen.tsx [hunk 2]
```diff
@@ -190,7 +190,7 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                        * We need to pass in the formatting for the phone number string in the onChange handler
                        * to make sure the phone number is in E.164 format.
                        */
-                      const phoneNumber = '+' + newValue?.replace('[^0-9]', '');
+                      const phoneNumber = '+' + newValue?.replace(/\D/g, '').replaceAll(' ', '');
```
Comment (anjulgarg) [inline samples/Calling/src/app/views/HomeScreen.tsx:193]: \D will take care of blank spaces as well :)

**Review by anjulgarg — APPROVED** [review]

File: samples/CallWithChat/src/app/views/HomeScreen.tsx [hunk 3]
```diff
@@ -139,7 +139,7 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                          *
                          * write regexp to flatten this
                          */
-                        const phoneNumber = '+' + newValue?.replace('[^0-9]', '');
+                        const phoneNumber = '+' + newValue?.replace(/\D/g, '');
```
Comment (carocao-msft) [inline samples/CallWithChat/src/app/views/HomeScreen.tsx:142]: probably need to remove spacing here? replace(' ', '')

Comment (dmceachernmsft) [reply samples/CallWithChat/src/app/views/HomeScreen.tsx:142]: this clears the spaces as well in my testing

**Review by carocao-msft — APPROVED** [review]

**Pull request merged**